### PR TITLE
Solve https://github.com/DiUS/java-faker/issues/707.

### PR DESCRIPTION
**Pull request by xiye2333333**
Please check it and notice us if there is any problem. Thanks a lot.
---- SE_SUSTech, group: Lanrand

### CODE REVIEW

**Comment by xiye2333333** [comment]
@bodiam Please check it and notice us if there is any problem. Thanks a lot.

**Comment by bodiam** [comment]
Awesome, thanks for the PR. There's a few small issues with the PR. If you could address them that would be great,  otherwise happy to accept and I'll fix it myself.

I'd prefer it if you would rename the NPC method to npc (so, lowercase). That's a bit more in line with Java conventions.

Also, the yml file has quite a few strings in all caps, and I was wondering if it would be possible to turn them into capital + lowercase, so APPLE becomes Apple. 

Otherwise, no issues at all, and thank you for even updating the Readme!

**Comment by xiye2333333** [comment]
@bodiam I use uppercase because the name in wiki (https://eldenring.wiki.fextralife.com/NPCs) are all in uppercase, but I think it's totally fine and better to convert them to lowercase. Thanks!

**Comment by bodiam** [comment]
I think that's just the layout on the page, if you click on the characters, for example https://eldenring.wiki.fextralife.com/Boc+the+Seamster, you see that the name is no longer all caps there, which I think would be more appropriate, and more in line with the other yml files we have.

**Comment by xiye2333333** [comment]
Ok, I get it. Thank you a lot!

**Comment by xiye2333333** [comment]
I have changed the letters to lowercase！

File: src/main/java/net/datafaker/EldenRing.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    public String skill(){
+        return faker.resolve("elden_ring.skill");
+    }
+
+    public String spell(){
+        return faker.resolve("elden_ring.spell");
+    }
+
+    public String NPC(){
```
Comment (bodiam) [inline src/main/java/net/datafaker/EldenRing.java:26]: Please change this to npc (lowercase)

File: src/main/resources/en/elden_ring.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+en:
+  faker:
+    elden_ring:
+      location: ["Limgrave","Weeping Penisula", "Liurnia Of The Lakes","Caelid","Altus Plateau","MT.Gelmir","Dragonbarrow","Mountaintops Of The Giants","Consecrated Snowfield"]
+      weapon: ["Dagger","Cinquedea","Short Sword","Sword of Night and Flame","Claymore","Dark Moon Greatsword","Grafted Blade Greatsword","Rapier","Carian Glintstone Staff"]
+      skill: ["Lion's Claw","Impaling Thrust","Piercing Fang","Spinning Slash","Charge Forth","Stamp","Blood Tax","Repeating Thrust","Wild Strikes","Spinning Strikes","Double Slash","Prelate's Charge"]
+      spell: ["Adula's Moonblade","Cannon Of Haima","Collapsing Stars","Comet","Glintstone Cometshard","Great Oracular Bubble","Loretta's Mastery","Ranni's Dark Moon"]
+      NPC: ["Sorceress Sellen","Smithing Master Hewg","Sorcerer Rogier","D","Rennala","Renna","Fia","Irina","Queen Marika"]
```
Comment (bodiam) [inline src/main/resources/en/elden_ring.yml:8]: Lowercase here too please

File: src/test/java/net/datafaker/EldenRingTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        assertThat(faker.eldenRing().skill()).matches("[A-Za-z' ]+");
+    }
+
+    @Test
+    public void spell() {
+        assertThat(faker.eldenRing().spell()).matches("[A-Za-z' ]+");
+    }
+
+    @Test
+    public void NPC() {
```
Comment (bodiam) [inline src/test/java/net/datafaker/EldenRingTest.java:29]: Lowercase please

**Review by bodiam — CHANGES_REQUESTED** [review]

File: src/main/resources/en/elden_ring.yml (hunk 2 above)
Comment (Lance-Azrael) [reply src/main/resources/en/elden_ring.yml:8]: Do we need to lower all the letters include names?

**Comment by bodiam** [comment]
Ah, sorry for confusing you : no, the values are fine, it's just the key (NPC) which needs to be lower case (npc)

**Comment by xiye2333333** [comment]
I have changed NPC to npc!

**Review by bodiam — APPROVED** [review]

**Comment by bodiam** [comment]
Awesome. Thanks for implementing these suggestions, looks great to me now! Approved, and thanks again for contributing, much appreciated!

**Comment by xiye2333333** [comment]
Thank you very much!

**Pull request merged**